### ISSUE TITLE
How to run the website demo locally?

### ISSUE THREAD

**Issue by not-my-profile** [pygments/pygments#1945]
It seems like Pyodide is currently only built in the GitHub action via a custom docker image.
Is there a way to run this locally? For example if you wanted to improve the demo.

**Comment by birkenfeld** [comment]
Yeah, there is no good description for that currently. You can probably see from the action that the "magic" all happens in the `birkenfeld/pyodide-pygments-builder` docker container, which is built from https://github.com/pygments/pyodide-artifacts. This then has to be run with the GITHUB_WORKSPACE variable set properly, but that should be more or less it.

**Comment by not-my-profile** [comment]
What should `GITHUB_WORKSPACE` be set to?

**Comment by birkenfeld** [comment]
If I interpret it correctly, it will be the path where the repository checkout is mounted into the container. The container entrypoint will copy the package from there, build the JS artifacts, and copy it back to `pyodide` in the checkout root. The workflow then copies that into the Sphinx output under `_static`.

**Comment by Anteru** [comment]
Should we keep this open? Do we need to update the docs?

**Comment by not-my-profile** [comment]
I actually did not manage to get it working with docker and ended up downloading the `_static/pyodide/` files from the [pygments.github.io](https://github.com/pygments/pygments.github.io) repository (which suffices for just testing changes to the demo JavaScript).
